src/strategy/spid.ts

```diff
@@ -133,7 +140,10 @@ export class SpidStrategy extends SamlStrategy {
       const samlService = new CustomSamlClient(
         {
           ...this.options,
-          ...samlOptions
+          ...samlOptions,
+          additionalParams: {
```

Comment:
why we need this one ? (does not `...this.options` suffice ?)

Reply:
If `additionalParams` are present inside `samlOptions` will override `additionalParams` inside `this.options`. This code prevent it.